cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,92 @@
 /// <reference types='cypress' />
+const { generateUser } = require('../support/generateUser');
 
 describe('Web Tables page', () => {
-  it('', () => {
+  let user;
+  const { firstName, lastName, email, age, salary, department } =
+  generateUser();
 
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  it('should have pagination', () => {
+    cy.get('.-pagination').should('exist');
+    cy.contains('.-next > .-btn', 'Next').should('exist');
+    cy.contains('.-previous > .-btn', 'Previous').should('exist');
```

Comment:
I would add checks if the "Next" and "Previous" buttons work (you add 3 new workers and buttons become active...)